Source/RxSwift/merge.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  merge.swift
```

Comment:
We should call this file **mergeWith.swift** to align with **zipWith.swift** IMO.

Reply:
Fixed, agree